lib/v2/domain-shared/page_command.dart

```diff
@@ -1 +1,33 @@
-class PageCommand {}
+abstract class PageCommand {}
```

Comment:
These are the base page commands, 
We can add any page commands that you see repeating in the app to this base.

Reply:
👏 👏 👏